website/src/scripts/setupSolidityDocs.ts

```diff
@@ -0,0 +1,38 @@
+const fileS = require("fs");
+const pathX = require("path");
+
+const solidityDocDir = pathX.join(__dirname, "../pages/solidity-docs");
```

Comment:
Hmmmm not sure how I feel about dropping these directly in the /pages/ directory of our source code. Is there not a way to drop these in a /static/ directory, like we do the for TypeDoc files? Or somehow just have them in the final build vs. within /src/?

Reply:
If we must do this, we should at least add this directory to .gitignore. Otherwise all these files show up as untracked files after running `npm run build`

Reply:
It also feels like a weird pattern that running a script in the `contracts` dumps a bunch of files in `website`... maybe it'd be cleaner to add a step?
1. `npm run docs` from `contracts` generates the static docgen output somewhere in the package
2. `npm run setup-soliditydocs` from `website` copies over that static directory?

Reply:
> It also feels like a weird pattern that running a script in the `contracts` dumps a bunch of files in `website`... maybe it'd be cleaner to add a step?
> 
> 1. `npm run docs` from `contracts` generates the static docgen output somewhere in the package
> 2. `npm run setup-soliditydocs` from `website` copies over that static directory?

I like this, good suggestion!

Reply:
> Hmmmm not sure how I feel about dropping these directly in the /pages/ directory of our source code. Is there not a way to drop these in a /static/ directory, like we do the for TypeDoc files? Or somehow just have them in the final build vs. within /src/?

I'll see if we can host them from within the artifacts, read that for markdown files the best way was to host within the pages folder - if that becomes too convoluted, could we go with the .gitignore trick?

Reply:
Looks like markdown files inside the static folders are not rendered, so it would just be the raw markdown. Gonna resort to adding the folder to .gitignore for now.

Reply:
> I'll see if we can host them from within the artifacts, read that for markdown files the best way was to host within the pages folder - if that becomes too convoluted, could we go with the .gitignore trick?

Ah, I see, and TypeDocs is in /static/ because they're HTML files?

Reply:
Correct! one option could be to put them in static and host them via some sort of React markdown library but think that would just make it slower/less responsive